code/modules/mob/living/living.dm

```diff
@@ -485,6 +485,10 @@
 	if ((!(L.stat) && !(L.restrained())))
 		resist_grab(L) //this passes L because the proc requires a typecasted mob/living instead of just 'src'
 
+	// Sliding out of a morgue/crematorium
+	if(loc && (istype(loc, /obj/structure/morgue) || istype(loc, /obj/structure/crematorium)))
+		resist_tray(L)
```

Comment:
this doesn't belong here, all carbon-specific resist procs should go under the iscarbon check